library/ui-styles/build.gradle

```diff
@@ -58,5 +58,5 @@ dependencies {
     // Pref theme
     implementation libs.androidx.preferenceKtx
     // dialpad dimen
-    implementation 'im.dlg:android-dialer:1.2.5'
```

Comment:
`im.dlg:android-dialer:1.2.5` is now replaced by the local module

Reply:
I guess we could avoid having this dependency now, but this is not really a big deal. Same for the String resource, they could be extracted from the module `dialpad` module.